base/applications/mspaint/precomp.h

```diff
@@ -3,10 +3,7 @@
 
 #include <stdarg.h>
 
-#include <windef.h>
-#include <winbase.h>
-#include <winuser.h>
-#include <wingdi.h>
+#include <windows.h>
```

Comment:
Please don't just go around changing these, this was done for build performance reasons.

Reply:
Reverted.